docs/pages/features/ssh-pam.mdx

```diff
@@ -1,334 +0,0 @@
----
```

Comment:
have we set up redirect here for this page?

Reply:
Yes - https://goteleport.com/docs/features/ssh-pam/ will redirect to the new article. I think this got merged into master during a merge or rebase. It was removed previously. In any event, it's harmless but extraneous.